tests/test_process_domain_events.py

```diff
@@ -51,3 +58,4 @@ def test_process_domain_events(empty_handler):
 
     # Assert
     assert empty_handler.handled_domain_events == [domain_event]
+    assert empty_handler.handled_many_domain_events == [[domain_event]]
```

Comment:
Please check that multiple events are group into a single list.

Reply:
today I will create PR and add this tests

Reply:
https://github.com/pristupa/ddd/pull/9